### ISSUE TITLE
Customer update with VerifyCard: false returns "Gateway Rejected: cvv"

### ISSUE THREAD

**Issue by ghost** [lionelbarrow/braintree-go#179]
Hi, I've been using braintree-go and I've come into an issue where I'm trying to update a customer's credit card billing address without using a payment method nonce, and I'm getting the `"Gateway Rejected: cvv"` error response.

I have the `VerifyCard` field set to `false` in `CreditCardOptions`, which I believe should help me avoid getting this error response.

Anyways, my code is at the bottom.  Let me know if anyone else has seen this same issue or if I'm doing anything incorrectly.

```go
c, err := s.api.Customer().Update(
	&braintree.Customer{
		Id: req.UserToken,
		CreditCard: &braintree.CreditCard{
			Options: &braintree.CreditCardOptions{
				UpdateExistingToken: req.PaymentMethodToken,
				VerifyCard:          false,
			},
			CardholderName: req.CardholderName,
			BillingAddress: &braintree.Address{
				StreetAddress:     req.BillingAddress.StreetAddress,
				ExtendedAddress:   req.BillingAddress.ExtendedAddress,
				PostalCode:        req.BillingAddress.PostalCode,
				CountryCodeAlpha2: req.BillingAddress.CountryCodeAlpha2,
			},
		},
	},
)
```

**Comment by leighmcculloch** [comment]
I tried to replicate the above on my sandbox account, but in every case I did not get the `Gateway Rejected: cvv` error that you did. It's possible that there's a setting configured in your account that's different to mine though that might cause verification to kick in. The `VerifyCard` parameter is only sent to the API if it is `true`. Setting it to `false` is the same as not setting it.

**Comment by esrobinson** [comment]
Yeah, I think this happens when the account is set up to verify card by default.  In that case, to add an address or update a card without providing the CVV, you have to pass `VerifyCard` as `false` to the API.

**Comment by leighmcculloch** [comment]
@ghost @esrobinson: Does #190 address the issue you're seeing?

**Comment by esrobinson** [comment]
Yeah, I think that takes care of it.  Thanks!

**Comment by leighmcculloch** [comment]
Fixed is released in `v0.15.0`.
